### PR TITLE
FIX:When opening or autoloading wallets there should be clear messages about rescanning in progress and wallets' names.

### PR DESCRIPTION
**Pull request by xonx4l**
issue-: #259

The "Opening Wallet" popup would now show "Opening Wallet: " instead of just "Opening Wallet". This identifies the specific wallet.

The "Rescanning" popup would show "Rescanning: " or "Rescanning - 0%" if a progress bar is added. Again, this identifies the wallet.

The main window title would include the wallet name after being opened/loaded thanks to the change in WalletView::showNormalIfMinimized().

Other dialogs like Send/Receive coins would also show the wallet name in the title.

So in summary, by appending the wallet name retrieved from WalletModel::getWalletName() to titles and popup messages, it provides clearer feedback to users about which wallet is being loaded and rescanned. No more confusion about which wallet Bitcoin Core is working on.

### CODE REVIEW

File: src/qt/receivecoinsdialog.cpp [hunk 1]
```diff
@@ -72,6 +72,9 @@ void ReceiveCoinsDialog::setModel(WalletModel *_model)
 {
     this->model = _model;
 
+        setWindowTitle(tr("Request payments - %1").arg(model->getWalletName()));
```
Comment (pablomartin4btc) [inline src/qt/receivecoinsdialog.cpp:75]: CI lint is [failing](https://github.com/bitcoin-core/gui/pull/786/checks?check_run_id=20356215277) due to unnecessary white-spaces in the files (you can run the script `lint-whitespace.py` locally or even [all linters](https://github.com/bitcoin/bitcoin/blob/master/test/lint/README.md#running-locally))
```suggestion
    setWindowTitle(tr("Request payments - %1").arg(model->getWalletName()));
```

**Review by pablomartin4btc — COMMENTED** [review]
Thanks @xonx4l for working on this.

I don't think this PR fixes #259 as, according to its description and what I understand, a fix should be add more clarification (percentage?) about the rescanning of a wallet when they get open in 2 situations:
1. During bitcoin-qt startup, as author of the PR specified, the user could add the wallets manually into the `settings.json` `qt` config file. In this case, perhaps the name of each wallet (if there are many, and may be also x/ total) should be added after the "rescanning..." text on the spash-screen (initial/ startup window).
2. When a user open a wallet for the first time thru the menu->File->Open Wallet. For this case, perhaps a "rescanning..." (and maybe a % or progress bar - check issue #679) text could be added at the bottom of the opening window.

@hebasto on his [comment](https://github.com/bitcoin-core/gui/issues/259#issuecomment-811297476) on the issue, mentioned to add the wallet name on the title of "Receiving addresses" & "Sending addresses" windows (which has been done and merged already on #757), not to the `receivecoins` or `sendcoins` dialogs which this PR changes, moreover since these dialogs are within the main window and titles won't be displayed:

image

Other than that, the title and description of the PR doesn't reflect the proper change that's making, please check the [contributor](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md) and [developer](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md) guidelines (e.g. commit name may start with prefix `gui:`, please check other PRs as reference).

Please don't get discouraged by these observations. Feel free to reach out if you have questions or need clarification on any points. I encourage you to continue working on this issue or any others. Thanks again!

**Comment by hebasto** [comment]
Closing due to lack of support from the PR author (unaddressed comments, failed CI).

Feel free to re-open.
